dss/storage/validator.py

```diff
@@ -0,0 +1,78 @@
+"""Utilities in this file are used to removed extra fields from index data before adding to the index."""
+import re
+from jsonschema.validators import create, RefResolver
+from jsonschema import _utils, _validators
+
+import json
+from urllib.request import urlopen
+from urllib.error import URLError
+import ssl
+from typing import List
+import logging
+
+
+def read_json_url(link):
+    gcontext = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+    f = urlopen(link, context=gcontext)
+    return json.load(f)
+
+
+def extract_json_fields(json_data: dict, path: List[str], fields: List[str]):
+    """
+        json_data: The json data from which to remove the field
+        path: The path through the json data to find the field.
+        fields: A list of fields names to remove from the json data at the locations specified by path."""
+    current = json_data
+    for next in path:
```

Comment:
`next` is a built-in and should be avoided.

Reply:
changed to `step`